app/components/certificate/description.tsx

```diff
@@ -1,18 +1,25 @@
-import { Flex, Heading, Text, HStack, VStack } from '@chakra-ui/react';
+import { RepeatIcon } from '@chakra-ui/icons';
+import { Flex, Heading, Text, HStack, VStack, IconButton } from '@chakra-ui/react';
+import { Form } from '@remix-run/react';
+import { useState } from 'react';
 
 interface DescriptionSectionProps {
   certRequested: boolean;
   validFromFormatted?: string;
   validToFormatted?: string;
   description: string;
+  canRenew?: boolean;
 }
 
 export default function DescriptionSection({
   certRequested,
   validFromFormatted,
   validToFormatted,
   description,
+  canRenew,
 }: DescriptionSectionProps) {
+  const [isDisabled, setIsDisabled] = useState(canRenew ? false : true);
```

Comment:
This is an anti-pattern in React.  You don't want to use props as the value of state, since the prop will change, and leave the state out of sync.  Below, use `canRenew ? false : true` as your value for the `disabled` attribute.